### PR TITLE
More instructions

### PR DESCRIPTION
**Pull request by hkwheat**
- [ ] Update https://github.com/utdscheld/name/wiki/Instruction-Support/ upon merging

### CODE REVIEW

**Review by CharlesAverill — APPROVED** [review]
LGTM

**Review by CowTipper989 — APPROVED** [review]
Looks good to me

File: name-emu/src/mips.rs [hunk 1]
```diff
@@ -146,6 +163,20 @@ impl Mips {
         let memory_address = (ins.rt as i64 + (ins.imm as i64)) as u32;
 
         match ins.opcode {
+            // Set Less Than Immediate (signed)
```
Comment (qwe-q) [inline name-emu/src/mips.rs]: self.regs[ins.rt] = if self.regs[ins.rs] < ins.imm as i8 as i32 as u32 { 1 } else { 0 }

**Comment by CharlesAverill** [comment]
Update https://github.com/utdscheld/name/wiki/Instruction-Support/ upon merging

File: name-emu/src/mips.rs (hunk 1 above)
Comment (hkwheat) [reply name-emu/src/mips.rs]: Done

**Review by qwe-q — APPROVED** [review]

**Pull request merged**